src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs

```diff
@@ -530,10 +530,8 @@ private async Task GatherOrganisationStandards(RegisterViewAndEditOrganisationVi
                 viewAndEditModel.RegisterViewOrganisationStandardsViewModel.OrganisationStandards.PaginationViewModel = await GatherOrganisationStandards(viewAndEditModel.OrganisationId, paged);
                 
                 viewAndEditModel.RegisterViewOrganisationStandardsViewModel.InProgressApplications.OrganisationId = viewAndEditModel.OrganisationId;
-                viewAndEditModel.RegisterViewOrganisationStandardsViewModel.InProgressApplications.PaginationViewModel = await GatherOrganisationStandardApplications(viewAndEditModel.OrganisationId, ApplicationReviewStatus.InProgress);
```

Comment:
Should we not be removing the properties from RegisterViewOrganisationStandardsViewModel?

Reply:
RegisterViewOrganisationStandardsViewModel is in use, GatherOrganisationStandardApplications method call is restored